Makefile

```diff
@@ -6,7 +6,7 @@ BINARY=rhc_catalog_worker
 .DEFAULT_GOAL := build
 
 build:
-	go build -ldflags="-X 'main.Version=${VERSION}' -X main.Sha1=`git rev-parse HEAD`" -o ${BINARY} ${SRC_FILES}
+	go build -ldflags="-X 'github.com/RedHatInsights/rhc-worker-catalog/build.Version=${VERSION}' -X github.com/RedHatInsights/rhc-worker-catalog/build.Sha1=`git rev-parse HEAD`" -o ${BINARY} ${SRC_FILES}
```

Comment:
@bzwei Should we create variables for the VERSION, BUILD etc that are built from the output of the git commands so they are not deeply nested